<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Baseline</class>
 <widget class="QWidget" name="Baseline">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>基线预览</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../QtWidgetsApplication3.qrc">
    <normaloff>:/QtWidgetsApplication3/bin/icon/InSARapp.ico</normaloff>:/QtWidgetsApplication3/bin/icon/InSARapp.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,7">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>目标工程</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,7">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>数据节点</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_dst_node"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,7">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>主图像选择</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_masterImage"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../QtWidgetsApplication3.qrc"/>
 </resources>
 <connections/>
</ui>
